docs/kb/integrations/pagination.md

```diff
@@ -0,0 +1,114 @@
+---
+slug: pagination
+title: How to paginate responses from the Semgrep API
+hide_title: true
+description: Learn how to paginate responses from the Semgrep API.
+tags:
+  - Deployment
+---
+
+# How to paginate responses from the Semgrep API
+
+Semgrep's API endpoints use both offset-based pagination and cursor-based pagination.
+
+## Offset-based pagination
+
+The following API endpoints support offset-based pagination:
+
+- [List all projects](https://semgrep.dev/api/v1/docs/#tag/Project/operation/semgrep_app.saas.handlers.repository.openapi_list_recent_projects)
+- [List code or supply chain findings](https://semgrep.dev/api/v1/docs/#tag/Finding/operation/semgrep_app.core_exp.findings.handlers.issue.openapi_list_recent_issues)
+
+For these endpoints, include the following query parameters to paginate through results:
+
+| Query parameter | Type | Description |
+| - | - | - |
+| `page` | integer | The page of results to return. Page numbering begins at `0`. Default: `0`|
+| `page_size` | integer | The maximum number of records returned per page. Default: `100`. |
+
+### Example
+
+To request a list of Code or Supply Chain findings, specifically the second page where each page contains 100 items, make a cURL call as follows:
+
+```console
+curl 'https://semgrep.dev/api/v1/deployments/docs_test/findings?page=2&page_size=100' \
+--header 'Authorization: Bearer YOUR_API_TOKEN'
+```
+
+## Cursor-based pagination
+
+The [List Secrets](https://semgrep.dev/api/v1/docs/#tag/SecretsService/operation/semgrep_app.products.secrets.handlers.issue.list_issues_conexxion) endpoint support cursor-based pagination:
```

Comment:
```suggestion
The [List Secrets](https://semgrep.dev/api/v1/docs/#tag/SecretsService/operation/semgrep_app.products.secrets.handlers.issue.list_issues_conexxion) endpoint supports cursor-based pagination:
```
"List Secrets" is the name of the endpoint, right? So it's a single endpoint and not plural?